### DISCUSSION TITLE
equivalent diameter

### DISCUSSION THREAD

**Discussion by yairmau** [PMEAL/porespy#903] [Q&A]
When retrieving the pore equivalent diameter, does the algorithm take into consideration if the network was produced from a 2d or 3d image? In the following code
https://porespy-fork.readthedocs.io/en/latest/_modules/porespy/network_extraction/__getnet__.html
the line
`net['pore.equivalent_diameter'] = 2*((3/4*net['pore.volume']/sp.pi)**(1/3))`
implies that the diameter is computed assuming a sphere, not a disk. Is this always the case, regardless of image dimension (2d/3d)?
Thanks

**Comment by jgostick** [comment]
You are correct.  The algorithm is really meant for 3D images, and 2D is just for demonstrations.  You can just used the inscribed diameter and compute the equivalent one yourself if needed to be more accurate.
